src/style.css

```diff
@@ -1 +1,134 @@
-/* empty source */
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  /* margin: 32px 19px 40px; */
+  /* margin-top: 16px; */
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  color: #060b35;
+  margin-top: 40px;
+  /* margin: 0 16px;   */
+}
+
+.card__productCode {
+  font-weight: 400;
+  font-size: 10px;
+
```

Comment:
you seem to be missing a `line-height: 14px;` here - try this to pass the tests

Reply:
fixed :)